### DISCUSSION TITLE
Discussion forum

### DISCUSSION THREAD

**Discussion by jarikmarwede** [jarikmarwede/Start-Menu-Helper#49] [Announcements]
This forum is for sharing ideas of helpful features, asking questions about problems you are having or even sharing your cleaning configs.

I think the [Issues](https://github.com/jarikmarwede/Start-Menu-Helper/issues) should be reserved for actual features that are fully fledged out and can be worked on, while vague feature ideas are better shared here first. The same goes for bug reports. If you are not quite sure what the problem is and need help figuring it out, this is the right place. If the feature or bug is then specific enough to be worked on the discussion thread can be converted to an issue.

**Comment by rickir206** [comment]
Dear Forum members:  I am trying to install the drivers for a pair of BSFrance LoRa boards.  Following the directions on github, I installed the Zadig installer. By double clicking the reset button, the installer recognizes the board as "Feather 32u4 (interface 0)", not STM32 Bootloader as described in the github instructions.   In the "Driver box" of Zadig is usbser (v10.0.19041.3636).  Apparently, I get to choose either (1) WinUSB (v6.1.7600.16385), (2) libusb-win32 (v1.2.6.0) or (3) libusbK (v3.0.7.0) as the driver to install.

My question is:  which of these three drivers should I install?

The STM32 Virtual COM Port driver described in the github instructions in not on the list.

**Reply by jarikmarwede** [reply]
I am not sure whether you are just trolling, but this is the forum for "Start-Menu-Helper" and definitely not the right place for whatever drivers you are trying to install.
Try looking for their own forum.
